src/index.js

```diff
@@ -1,5 +1,23 @@
 export { mix, is, like }
 
+/**
+ * Accepts an optional superclass as the first argument,
+ * then a bunch of mixins and an optional class factory as the last argument and returns a mixin.
+ * Mostly, you will be using mix with a factory to create mixins, like this:
+ *
+ * var Looker = mix(superclass => class Looker extends superclass {
+ *   constructor() {
+ *     super()
+ *     console.info('A looker is born!')
+ *   }
+ *   look() {
+ *     console.info('Looking good!')
+ *   }
+ * })
+ *
+ * @param {function} args
```

Comment:
right, however the docs should reflect actual parameters, so you would have to change the source... However, I can just write it next to the param as comment